src/pageObject.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Header {
```

Comment:
Consider adding a constructor to the Header class for initializing the 'logo' property. This would make the class more flexible and allow the logo to be easily changed, if necessary.

Reply:
Added. Thanks